backend/models/user.py

```diff
@@ -0,0 +1,45 @@
+class User:
+    def __init__(self, FName, LName, email, location, role, PetId, avatar, userId=""):
```

Comment:
PetID is optional, so it should be PetID = []
I think Avatar is optional as well